src/pages/Editor.vue

```diff
@@ -320,7 +320,12 @@
       data-test="imageUploaderDialog"
```

Comment:
if we are going ahead with the proposed change in the logic for scorecard evaluation of subjective questions (as mentioned [here](https://www.notion.so/avantifellows/Scorecard-at-the-end-of-the-video-8d48ce456c4a4a0f92f2d6ffc1a14a83)), then need to change the info text that we show to creators that if anyone answers it, it is considered correct